### ISSUE TITLE
can we have "test baby" accounts to try things out?

### ISSUE THREAD

**Issue by ebergelson** [lookit/lookit-api#998]
# TL;DR
for testing out how an experiment feels on the participant side, having a "test kid" would be great

# Narrative
As experimenters, sometimes we want to run through a study like the participants would, and i think right now we just have to create a 'fake child' of whatever relevant age, but then that child will get emails asking them to participate in studies, etc. If instead there was some way that researcher accounts could have a 'test baby of age x' kid, this could be avoided

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ X ] Tests for new feature

# Implementation Notes
no strong feelings on implementation
